fake-ppud-automation-api/__files/search.json

```diff
@@ -1,12 +1,4 @@
 {
   "results": [
-    {
```

Comment:
Is deleting this data included by mistake?

Reply:
Eek!

Reply:
Re-introduced!